lib/whitespace.coffee

```diff
@@ -45,13 +46,17 @@ class Whitespace
   removeTrailingWhitespace: (editor, grammarScopeName) ->
     buffer = editor.getBuffer()
     scopeDescriptor = editor.getRootScopeDescriptor()
+    modifiedRows = @_getModifiedRows(editor)
```

Comment:
You might want to only call `getModifiedRows` if `onlyModifiedLines` is `true`. That way only people that care about the difference will pay the perf hit.